### PR TITLE
Unify webpack config

### PR DESCRIPTION
**Pull request by skjnldsv**
This will simplify the config and remove a lot of (bad?) duplicates.
This will also allow us to use global eslint config for specific filetypes (like vue) :)

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]
As discussed. Good move!

**Comment by skjnldsv** [comment]
Some errors that where I could use some help:
- [ ] /apps/files
  ```
  Uncaught TypeError: dav.Client is not a constructor
      at new Client (client.js:62)
      at Object.OC.Files.getClient (client.js:936)
      at Object.initialize (app.js:91)
      at app.js:344
      at underscore.js:829
  ```
- [ ] /settings/user/security
  ```
  Uncaught TypeError: Cannot read property 'Templates' of undefined
      at templates.js:2
      at templates.js:36
  ```
- [ ] /settings/user/security
  ```
  [Vue warn]: Property or method "actions" is not defined on the instance but referenced during render. Make sure that this property is reactive, either in the data option, or for class-based components, by initializing the property. See: https://vuejs.org/v2/guide/reactivity.html#Declaring-Reactive-Properties.
  
  found in
  
  ---> <AuthToken> at settings/src/components/AuthToken.vue
         <AuthTokenList> at settings/src/components/AuthTokenList.vue
           <Root>
  ```

**Comment by skjnldsv** [comment]
- [ ] /settings/admin/workflow
  ```
  backbone-webdav.js:255 Uncaught TypeError: davclient_js__WEBPACK_IMPORTED_MODULE_1___default.a.Client is not a constructor
      at Object.davCall (backbone-webdav.js:255)
      at child.sync (backbone-webdav.js:360)
      at child.fetch (backbone.js:1064)
      at child.fetch (systemtagscollection.js:77)
      at Object.init (admin.js:37)
      at HTMLDocument.<anonymous> (admin.js:176)
      at fire (jquery.js:3099)
      at Object.fireWith [as resolveWith] (jquery.js:3211)
      at Function.ready (jquery.js:3417)
      at HTMLDocument.completed (jquery.js:3433)
  ```

File: apps/workflowengine/webpack.js [hunk 1]
```diff
@@ -9,36 +9,13 @@ module.exports = {
...
-			},
-			{
-				test: /\.(png|jpg|gif|svg)$/,
-				loader: 'file-loader',
-				options: {
-					name: '[name].[ext]?[hash]'
-				}
-			},
 			{
 				test: /\.handlebars$/,
```
Comment (skjnldsv) [inline apps/workflowengine/webpack.js]: Not sure if we really need this helperDir thingy. @blizzz ?
What is is used for?

**Comment by skjnldsv** [comment]
Bump! All good! Failure unrelated!
Rebased and ready to go!

**Review by rullzer — APPROVED** [review]
CI is happy so lets do this

**Pull request merged**